conda_build/config.py

```diff
@@ -342,6 +342,23 @@ def target_subdir(self):
     def target_subdir(self, value):
         self._target_subdir = value
 
+    @property
+    def exclusive_config_file(self):
+        if self.exclusive_config_files:
+            return self.exclusive_config_files[0]
+        return None
+
+    @exclusive_config_file.setter
+    def exclusive_config_file(self, value):
+        if len(self.exclusive_config_files) > 1:
```

Comment:
Is it safe to assume `self.exclusive_config_files` is always a `list` or should we also support the case where someone sets it to `None`?

Reply:
Yes, I think this is a decent assumption.  We can wrap it with ensure_list if necessary.